docs/sidebar.json

```diff
@@ -47,7 +47,7 @@
     },
     {
       "type": "category",
-      "label": "Best Practices",
+      "label": "Design Guidelines",
```

Comment:
I'm not sure I agree with 'Design Guidelines' as a name. Ours are more general, not necessarily only about design. Lots of frameworks use 'Best Practices' as a name, my gold standard comparison is Cypress: https://docs.cypress.io/guides/references/best-practices